Gruntfile.js

```diff
@@ -55,41 +57,26 @@ module.exports = function(grunt) {
       },
     },
 
-    "closure-compiler": {
-      dist: {
-        files: {
-          "dist/browser-polyfill.min.js": ["dist/browser-polyfill.js"],
-        },
-        options: {
-          // Closure currently supports only whitespace and comment stripping
-          // when both the input and output languages are ES6.
-          compilation_level: "WHITESPACE_ONLY",
-          language_in: "ECMASCRIPT6_STRICT",
-          language_out: "ECMASCRIPT6",
-          output_wrapper: `${LICENSE}\n%output%`,
-        },
+    shell: {
+      minify: {
+        command: "babili dist/browser-polyfill.js -o dist/browser-polyfill.min.js --minified --no-comments --source-maps true",
       },
     },
 
-    // This currently does not support ES6 classes.
-    uglify: {
-      options: {
-        banner: LICENSE,
-        compress: true,
-      },
-
-      dist: {
-        files: {
-          "dist/browser-polyfill.min.js": ["dist/browser-polyfill.js"],
-        },
+    concat: {
+      license: {
+        src: "dist/browser-polyfill.min.js",
+        dest: "dist/browser-polyfill.min.js",
+        options: {footer: MINIFIED_FILE_FOOTER},
```

Comment:
Unfortunately there are two issues that I noticed by trying to use `auxiliaryCommentBefore` as suggested above:

- it didn't work if the babel is configured to do not output any comment (which is needed to exclude all the inline comments from the minified version)
 
- it didn't work as I expected initially (it doesn't add a single comment, but multiple comments, basically its "most common use case" is: including the "istanbul ignore next" on the code injected by babel that is not coming from the original transpiled code)